cli/internal/cmd/run.go

```diff
@@ -526,6 +537,9 @@ func newRunCreateCommandCore(
 
 	cmd.Flags().BoolVar(&flags.pull, "pull", false, "Pull container images. Applies only to Tyger running in Docker.")
 
+	cmd.Flags().StringVar(&flags.bufferAccessTtl, "buffer-access-ttl", "", "The time-to-live for each buffer's SAS URL (format D.HH:MM).")
```

Comment:
I think there should be some mention that this ttl in until the token is refreshed.